### ISSUE TITLE
implement learning rate finder

### ISSUE THREAD

**Issue by bwanglzu** [jina-ai/finetuner#689]
Find a proper learning rate is not easy. This is especially the case for client & Jina AI Cloud users.

we should implement the automatic learning rate finder as a callback function to tweek the appropriate learning rate.
